opengapps-packages.mk

```diff
@@ -159,12 +159,19 @@ PRODUCT_PACKAGES += \
     PrebuiltBugle
 endif
 
-ifeq ($(GAPPS_FORCE_PIXEL_HOME),true)
+ifeq ($(GAPPS_FORCE_PIXEL_LAUNCHER),true)
 GAPPS_EXCLUDED_PACKAGES += \
     GoogleHome
 
+ifeq (1,$(filter 1,$(shell echo "$$(( $(PLATFORM_SDK_VERSION) >= 25 ))" )))
+# never dexopt the PixelLauncherIcons
+$(call add-product-dex-preopt-module-config,PixelLauncherIcons,disable)
```

Comment:
You can specify this in the PixelLauncherIcons package. See:

https://source.android.com/devices/tech/dalvik/configure.html#local_dex_preopt